text/0003-browser-support.md

```diff
@@ -0,0 +1,45 @@
+# Project title
+- Start Date: 2019/04/26
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/3
+
+## Decision, including impact on distributions
+Core OpenMRS frontend code supports the following browsers:
+- Last 2 Safari major versions
+- Last 3 Edge major versions
+- Last 5 Chrome major versions
+- Last 3 Firefox major versions
+- Last 3 Opera major versions
+
```

Comment:
As context, at PIH we only officially support Chrome.  We don't code or test for any other browser types or versions and make no guarantee about how well things will work.  OpenMRS _may_ want to be more flexible than that, but given limited development and testing resources, I doubt we will every truly certify that all of those listed browsers are supported.

Reply:
AMPATH only supports chrome as well.

Reply:
The cost of supporting [evergreen browsers](https://www.techopedia.com/definition/31094/evergreen-browser) is very comparable to just supporting chrome. You can support firefox, safari, and edge with literally no work for the majority of code and projects.

I disagree in principle with only supporting chrome -- it sends a ton of usage data to Google servers and is scary if you care about privacy. It's definitely a nice browser and one that should be supported, but I think that saying "you must give all of your browsing history to Google so that they can advertise to you" is too strong of a stance for a large open source project.

My preference is to have OpenMRS support evergreen browsers, and let distributions decide on their own about what they'll support

Reply:
i have not tried them out but looks like there are some cross-browser testing tools we can use to reduce the testing resource requirements.

Reply:
👍 yeah i've used saucelabs before and it's alright. Generally speaking, though, much of the code we write will work in evergreen browsers without intensive testing, though.

Reply:
Big ups to supporting firefox, in light of the recent news about Chrome’s disabling adblockers...

Reply:
I agree with supporting evergreen browsers...